ReleaseNotes.md

```diff
@@ -11,6 +11,43 @@ additional questions or comments.
 
 Note : The agent version(s) below has dates (ciprod<mmddyyyy>), which indicate the agent build dates (not release dates)
 
+### 5/19/2022 -
+##### Version microsoft/oms:ciprod05192022 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod05192022 (linux)
+##### Version microsoft/oms:win-ciprod05192022 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-ciprod05192022 (windows)
+##### Code change log
+- Linux Agent
+  - PodReadyPercentage metric bug fix
+  - add cifs & fuse file systems to ignore list
+  - CA Cert Fix for Mariner Hosts in Air Gap
+- Windows Agent
+  - Ruby version upgrade from 2.6.5.1 to 2.7.5.1
+  - Added Support for Windows Server 2022
+  - Multi-Arch Image to support both Windows 2019 and Windows 2022
+- Common (Linux & Windows Agent)
```

Comment:
@jatakiajanvi12 , we also need to add this line under common "configmap updates for log collection settings for v2 schema"

Reply:
Updated!